tasks/sufia-user.rake

```diff
@@ -0,0 +1,14 @@
+namespace :sufia do
+  namespace :user do
+
+    desc "list user's email"
+    task "list_emails", [:file_name] => :environment do |cmd, args|
+      file_name = args[:file_name]
```

Comment:
@jcoyne since it is just a helper method it makes more sense to keep all the code in one place.